apps/public-docsite-v9/src/Concepts/SSR/SSR.stories.mdx

```diff
@@ -195,3 +195,13 @@ const DefaultOpenMenu = () => {
   );
 };
 ```
+
+### Strict mode
+
+Strict mode is currently not supported. To avoid hydration issues, disable strict mode in your app by adding the following configuration to your `next.config.js` file:
```

Comment:
Is this for strict mode in any version of React or only in React 18?

```suggestion
Strict mode in React 18 is currently not supported. If you're using React 18, to avoid hydration issues, disable strict mode in your app by adding the following configuration to your `next.config.js` file:
```

Reply:
This is also for React 17